go.mod

```diff
@@ -11,6 +11,6 @@ require (
 	google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8
 	google.golang.org/grpc v1.19.0
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
-	gopkg.in/resty.v1 v1.12.0
+	gopkg.in/resty.v1 v1.12.0 // indirect
```

Comment:
If you can just revert this we can merge this.

Reply:
It was generated by the first docker cmd - wasn't sure whether to commit it or not. I don't know anything about Go modules so wasn't sure why it did that or if that was a desired change or not. I'll revert if that is the desired state.